content/docs/command-reference/plots/show.md

```diff
@@ -137,6 +157,26 @@ $ dvc plots show logs.csv
 file:///Users/usr/src/plots/logs.csv.html
 ```
 
+<details>
+
+### Expand for TSV format
+
+Here's a corresponding `train.tsv` metrics file:
```

Comment:
Same here, just put both these blocks right after the JSON block code please...

Reply:
Done. But the TSV block would look better after CSV block code think.

Reply:
> TSV block would look better after CSV block

Wait that does sound logical, let me check... ⌛